v3/internal/templates/_common/Taskfile.tmpl.yml

```diff
@@ -310,17 +311,56 @@ tasks:
       DESKTOP_FILE: '{{ "{{.APP_NAME}}" }}.desktop'
       OUTPUT_DIR: '../../bin'
 
+  create:rpm:
+    summary: Creates a RPM package
+    dir: build/rpm
+    platforms: [ linux ]
+    deps:
+      - task: build:linux
+        vars:
+          PRODUCTION: "true"
+      - task: generate:linux:dotdesktop
+    cmds:
+      # Copy binary + icon to rpmbuild dir
+      - cp {{ "{{.APP_BINARY}}" }} {{ "{{.APP_NAME}}" }}
+      - cp ../appicon.png '{{ "{{.APP_NAME}}" }}'.png
+      - cp ../rpm/app.spec '{{ "{{.APP_NAME}}" }}'.spec
+      - sed -i -e 's/@NAME@/{{ "{{.APP_NAME}}" }}/g'
+              -e 's/@SUMMARY@/{{ "{{.APP_SUMMARY}}" }}/g'
+              -e 's/@VERSION@/{{ "{{.APP_VERSION}}" }}/g'
+              -e 's/@DESCRIPTION@/{{ "{{.APP_DESC}}" }}/g'
+              -e 's/@AUTHOR@/{{ "{{.APP_AUTHOR}}" }}/g'
+              -e 's/@AUTHOR_EMAIL@/{{ "{{.AUTHOR_EMAIL}}" }}/g'
+              {{ "{{.APP_NAME}}" }}.spec
+      # Generate RPM
+      - wails3 generate rpm -binary {{ "{{.APP_NAME}}" }} -icon {{ "{{.ICON}}" }} -desktopfile {{ "{{.DESKTOP_FILE}}" }} -outputdir {{ "{{.OUTPUT_DIR}}" }} -builddir {{ "{{.ROOT_DIR}}" }}/build/rpm
+      - mv {{ "{{.ROOT_DIR}}" }}/build/rpm/$(uname -m)/*.rpm {{ "{{.OUTPUT_DIR}}" }}
+    vars:
+      APP_NAME: '{{ "{{.APP_NAME}}" }}'
```

Comment:
I think it would be better to use the variables at the top of the Taskfile and add any new ones there.